files/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
This file would also be generated when not using yarn. I think we should filter this out [here](https://github.com/embroider-build/addon-blueprint/blob/main/index.js#L298-L327)!

Reply:
Thx for pointing this out @simonihmig . I'll update the PR

Reply:
Do you know why the default `pnp` setting does not work here? And is that something we could fix, instead of opting for `node-modules` here?

Reply:
By default, the `pnp` setup stores all packages in the `.yarn/cache` folder as `.zip` files.

The Ember build system, classic or Embroider, does not interact with these .zip files and needs the `node_modules` folder.

I believe that Yarn provides primitives to allow reading from `.yarn/cache/*.zip` as if it was an uncompressed/regular file system, and the Ember build system could leverage this to be compatible with `pnp`.

I tinkered a bit with it in the past, but it implied many changes and I did not finish the work… And then, pnpm kinda took over, and I figured it was not worth going this direction since we had the `nodeLinker: node-modules` workaround.